test/visibility_test/visibility_test.go

```diff
@@ -17,9 +17,14 @@
 package visibility_test
 
 import (
+	"fmt"
```

Comment:
a general question regarding testing labeling and field querying - every type will support field querying and most types will support labeling - so we would probably need to copy the same 1k lines of tests for each resource type ? Would it make sense to put labeling and field querying tests in a separate package and parameterize them so that we can just add the new api type when to want to add tests for either labeling or fieldquerying?

Reply:
i agree, but it would be better if it comes in a new PR where all entities begin to support labels